app/src/main/java/com/x8bit/bitwarden/data/autofill/fido2/model/Fido2GetCredentialsResult.kt

```diff
@@ -0,0 +1,26 @@
+package com.x8bit.bitwarden.data.autofill.fido2.model
+
+import androidx.credentials.provider.BeginGetPublicKeyCredentialOption
+import com.bitwarden.fido.Fido2CredentialAutofillView
+
+/**
+ * Represents the result of a FIDO 2 Get Credentials request.
+ */
+sealed class Fido2GetCredentialsResult {
```

Comment:
Isn't this the exact same model as `Fido2GetCredentialModel`? Do we need both?

Reply:
It is the same object. `Fido2GetCredentialModel` is being renamed in this PR. Looks like it got added back while updating from `main`. 👍